### DISCUSSION TITLE
Home Page Structure

### DISCUSSION THREAD

**Discussion by PleaseStopAsking** [LeoKlaus/plappa#5] [Feature Requests]
First off, thank you for the project, especially for the ABS support.

I wanted to open a new request to improve the home page structure to make it easier to navigate. The current UI has everything listed in a vertical format for each section. It would likely be easier cognitively, to understand the sections if they were oriented horizontally much in the manner that the official ABS app does or even Prologue.

If the vertical sections were a design decision, that is fine but the section headers may benefit from a large font to have them stand out as they sort of blend in on an iPhone 14 Pro at the moment.

**Comment by LeoKlaus** [comment]
I absolutely agree, I'll see to change this.

**Reply by PleaseStopAsking** [reply]
I got bored and decided to do a mockup in Xcode for fun.

<img width="342" alt="image" src="https://github.com/LeoKlaus/plappa/assets/35308946/b1cb49e8-7885-4ba6-8cd0-130728db64a0">

**Comment by PleaseStopAsking** [comment]
Just installed the latest release and the new horizontal scroll UI is nice. Thanks for the very quick turn around this.

The performance improvements made a big difference as well as I have a large library!

**Comment by LeoKlaus** [comment]
Glad you like it! If you have any other suggestions, feel free to let me know :)

I'll close this discussion for now.
